src/locales/rw/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,627 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2020-08-07 15:22+0200\n"
+"PO-Revision-Date: 2020-08-07 12:17+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: rw\n"
+"Language-Team: rw <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: services/messenger.py:40
+msgid "Friend"
+msgstr "Nshuti"
+
+#: services/messenger.py:75
+#, python-format
+msgid ""
+"🙏🏼 Hi %(first_name)s, so you’ve decided to make your first steps in Open "
+"Source. That’s great."
+msgstr ""
+"🙏🏼 Muraho% (first_name), nuko rero wahisemo gutera intambwe zawe za mbere"
```

Comment:
@Emile-Nsengimana bad translation makes sure to write `Muraho %(first_name)s, nuko....`